autoload/slime/common.vim

```diff
@@ -24,7 +24,13 @@ function! slime#common#write_paste_file(text)
   if !isdirectory(paste_dir)
     call mkdir(paste_dir, "p")
   endif
-  let output = slime#common#system("cat > %s", [slime#config#resolve("paste_file")], a:text)
+  if &l:shell ==# "cmd.exe"
+      let output = slime#common#system("type > %s", [slime#config#resolve("paste_file")], a:text)
+  elseif &l:shell ==# "powershell.exe"
+      let output = slime#common#system("echo > %s", [slime#config#resolve("paste_file")], a:text)
+  else
+      let output = slime#common#system("cat > %s", [slime#config#resolve("paste_file")], a:text)
+  endif
```

Comment:
the `cat >` always felt like a hack…

I've been experimenting with [this](https://github.com/jpalardy/dotfiles/blob/main/vim/mappings.vim#L114). Can you give this a try and let me know if that works for you.

(untested) something like:

```vim
call writefile([a:text], slime#config#resolve("paste_file"))
```

Reply:
I will give it a try, love it thanks!

Reply:
That line worked flawlessly! would recommend. I'll add it to my PR.